### DISCUSSION TITLE
Ability to add divider with <Stack/> component

### DISCUSSION THREAD

**Discussion by thevipinmishra** [mantinedev/mantine#2367] [Feature requests]
We usually need some sort of dividers/borders when working with `<Stack />`, it would be great to have a prop in which we can pass a divider component that can get added between stack items.

MUI has this implemented, for the inspiration. ( https://mui.com/material-ui/react-stack/#dividers )

![image](https://user-images.githubusercontent.com/52159805/188561929-c83fd950-c82c-4fad-88db-c84d00ebf34e.png)

**Comment by zhulien-ivanov** [comment]
This is useful for both `Group` and `Stack`. I was going to propose this myself. 

For API, maybe support a property like `splitBy` that accepts a `ReactNode` and joins the `Stack`/`Group` items?

**Reply by rtivital** [reply]
Yeah, I'm fine with it, mui props names seems to be reasonable
